rules/arrow-parens.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Rule to require parens in arrow function arguments.
+ * @author Jxck
+ * @copyright 2015 Jxck. All rights reserved.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+    var message = "Expected parentheses around arrow function argument.";
+    var asNeededMessage = "Unexpected parentheses around single function argument";
+    var asNeeded = context.options[0] === "as-needed";
+
+    /**
+     * Determines whether a arrow function argument end with `)`
+     * @param {ASTNode} node The arrow function node.
+     * @returns {void}
+     */
+    function parens(node) {
+        var token = context.getFirstToken(node);
+        if (node.async) token = context.getTokenAfter(token);
```

Comment:
added this line like I mentioned in https://github.com/babel/eslint-plugin-babel/issues/8#issuecomment-130103979